spacy/pipeline/callbacks.py

```diff
@@ -0,0 +1,20 @@
+from typing import Type, Callable
+
+from thinc.layers import with_nvtx_range
+from thinc.model import wrap_model_recursive
+
+from ..language import Language
+from ..pipeline import TrainablePipe
+from ..util import registry
+
+
+@registry.callbacks("spacy.models_with_nvtx_range.v1")
+def create_models_with_nvtx_range() -> Callable[[Type[Language]], Type[Language]]:
```

Comment:
I probably just don't understand the details, but: why is this `Type[Language]` instead of `Language`?

Reply:
You are right, this is not a callback that can be used for `before_creation`, so the type signature is wrong. Thanks, I have fixed this.

Reply:
Ah, okay. I think the return type on the callback I wrote in `spacy.training.callbacks` is also wrong, but in the other direction.